src/gpu/nvidia/sycl_cuda_engine.cpp

```diff
@@ -239,6 +239,10 @@ constexpr dnnl::impl::impl_list_item_t sycl_cuda_impl_list[] = {
         INSTANCE(cudnn_binary_t)
         INSTANCE(sycl::ref_binary_t)
 
+        //layer norm
+        INSTANCE(sycl::ref_layer_normalization_fwd_t)
```

Comment:
(random spot) Please add documentation for this primitive

Reply:
Yes. Added.